README.md

```diff
@@ -837,6 +837,7 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 
 #### Window Management
 
+* [AltTab](https://alt-tab-macos.netlify.app) - Open source window switcher with window previews. [![Open-Source Software][OSS Icon]
```

Comment:
Open-Source URL: https://github.com/lwouis/alt-tab-macos @mnin

Reply:
@jaywcjlove would you like add both URLs here? Thought, one should be enough and the given explain it better than the GitHub project URL.

Reply:
@jaywcjlove done

Reply:
@mnin thx!